### PR TITLE
Update cypress version to 2.0.2 - Closes #563

### LINKED ISSUE #563: Update cypress dependency
We're still using a 0.x version, but v1.x is out.

### PR DESCRIPTION
**Pull request by shuse2**
Closes #563

### CODE REVIEW

**Comment by willclarktech** [comment]
@shuse2 Can you correct the spelling in the commit message?

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+module.exports = () => {
+	// module.exports = (on, config) => {
+	// `on` is used to hook into various events Cypress emits
+	// `config` is the resolved Cypress config
+};
```
Comment (willclarktech) [inline cypress/plugins/index.js]: Do we need this file?

Comment (shuse2) [reply cypress/plugins/index.js]: It is auto generated, so I think it would be better to keep it.

Comment (willclarktech) [reply cypress/plugins/index.js]: Why not add it to `.gitignore` and add it to vcs when necessary (if ever)?

Comment (shuse2) [reply cypress/plugins/index.js]: sounds good to me 👍

**Pull request merged**